app/schema/portal/cdo/tasks/record-verification-dates.js

```diff
@@ -2,38 +2,75 @@ const Joi = require('joi')
 const { validateDate, getDateComponents } = require('../../../../lib/date-helpers')
 const { validatePayloadBuilder } = require('../../../../schema/common/validatePayload')
 
+const microchipVerification = Joi.object({
+  year: Joi.string().allow(null).allow(''),
+  month: Joi.string().allow(null).allow(''),
+  day: Joi.string().allow(null).allow('')
+}).required().custom((value, helper) => validateDate(value, helper, true, true))
+  .messages({
+    'any.required': 'Enter a microchip number verified date'
+  })
+
+const neuteringConfirmation = Joi.object({
+  year: Joi.string().allow(null).allow(''),
+  month: Joi.string().allow(null).allow(''),
+  day: Joi.string().allow(null).allow('')
+}).required().custom((value, helper) => validateDate(value, helper, true, true))
+  .messages({
+    'any.required': 'Enter a neutering verified date'
```

Comment:
Not quite sure of the correct wording. Should it be 'Enter a neutering confirmed date' ??